### PR TITLE
Use templates and external config for template values

### PR DESCRIPTION
**Pull request by omh1280**
Use [`handlebars`](http://handlebarsjs.com/) to clean up the messages using templating. The variable strings will eventually move into a configuration file and out of the source code.

What is the best way to specify the configurable strings like `signature` and `project`? Should they be in a `.js` file that looks like:

```javascript
var signature = 'blah';
var project = 'cesium';
...
```

Or should they be in `config.json`?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
@omh1280 thanks for the pull request!

I noticed that [`CHANGES.md`](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the Cesium API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and bump this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

__I am a bot who helps you make Cesium awesome!__ Thanks again.

**Comment by omh1280** [comment]
Not quite sure how to fix the `create-zip` error that is occuring

**Comment by mramato** [comment]
> What is the best way to specify the configurable strings like signature and project?

Each message should be a single per-repository configuration setting just like everything else.  You can put the defaults in external files if it makes it easier to manage them, but users that override them will just specify them along with any other configuration.

**Comment by omh1280** [comment]
This is now grabbing the values to fill the template with from each repository's configuration. This is the cleanest way I could think to do it (pass them as `templateObject` from `postToGitHub.js`).

There are currently no defaults. The way I could add defaults is by checking each repository for every `key` that we expect to be set, and populating it if it's not there. I can do that if you think it's the best way to do it.

**Comment by mramato** [comment]
> There are currently no defaults. The way I could add defaults is by checking each repository for every key that we expect to be set, and populating it if it's not there. I can do that if you think it's the best way to do it.

We definitely want defaults for everything, the pre-populate strategy is probably okay.  It's not the most memory efficient, but it should be simple to do.  If you keep all of the comments in a single object, it should be a single statement to copy them over.

File: bin/stalePullRequest.js [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
 var defined = Cesium.defined;
 
-var dateLog = require('../lib/dateLog');
 var checkStatus = require('../lib/checkStatus');
+var dateLog = require('../lib/dateLog');
 var Settings = require('../lib/Settings');
 
 module.exports = stalePullRequest;
 
+var firstMessage = 'Thank you for the pull request!\n\n' +
```
Comment (mramato) [inline bin/stalePullRequest.js]: I'm confused as to why these are still here.  I would expect the defaults to be in Settings.

**Comment by pjcozzi** [comment]
Fixes #53?

**Pull request closed without merging**